src/__tests__/posthog-core.js

```diff
@@ -141,6 +141,7 @@ describe('capture()', () => {
     )
 
     given('overrides', () => ({
+        __loaded: true,
```

Comment:
Without this change, these test would fail.  The only reason they actually were not failing before, was because `_captureMetrics` was being specifically overridden a few lines down.